packages/engine-core/src/data-proxy/DataProxyEngine.ts

```diff
@@ -55,12 +55,6 @@ export class DataProxyEngine extends Engine {
     this.host = host
 
     debug('host', this.host)
-
-    if (this.config.previewFeatures?.includes('tracing')) {
```

Comment:
maybe replace with a mention of packages/internals/src/get-generators/utils/check-feature-flags/checkFeatureFlags.ts, as this seems to be a natural location to do this manually instead?